m4/ax_python.m4

```diff
@@ -186,7 +186,8 @@ AC_REQUIRE([PKG_PROG_PKG_CONFIG])dnl
       AC_MSG_RESULT([yes])
 
       PYTHON_CFLAGS="`\"$pycfg\" --includes`"
-      if test x"${py_ver}" = x"3.8" || test x"{py_ver}" = x"3.9"; then
+      minor_ver=${py_ver#*\.}
+      if test $((minor_ver)) > 7; then
```

Comment:
I have no idea if this syntax is valid in m4, but I get the same result regardless of what I try (in bash), and not so for the previous comparison:
```
qlyoung@nicobar:~$ test x"8" = x"8"
qlyoung@nicobar:~$ echo $?
0
qlyoung@nicobar:~$ test x"8" = x"9"
qlyoung@nicobar:~$ echo $?
1
```
versus
```
qlyoung@nicobar:~$ py_ver=3.8
qlyoung@nicobar:~$ minor_ver=${py_ver#*\.}
qlyoung@nicobar:~$ echo $minor_ver
8
qlyoung@nicobar:~$ test $((minor_ver)) > 7
qlyoung@nicobar:~$ echo $?
0
qlyoung@nicobar:~$ test $((minor_ver)) > 9
qlyoung@nicobar:~$ echo $?
0
```
However, I get a correct result with `-ge`:

```
qlyoung@nicobar:~$ test $((minor_ver)) -ge 9
qlyoung@nicobar:~$ echo $?
1
qlyoung@nicobar:~$ test $((minor_ver)) -ge 6
qlyoung@nicobar:~$ echo $?
0
```
Can you verify that this test actually works?